docker/Dockerfile.amber

```diff
@@ -0,0 +1,17 @@
+FROM rust:1.67 as builder
```

Comment:
It would be of great help to have a document for Amber integration and this `Dockerfile`

Reply:
I added some documents in the README.md